src/java/htsjdk/samtools/cram/common/CramVersions.java

```diff
@@ -7,5 +7,5 @@
     /**
      * The default CRAM version when creating a new CRAM output file or stream.
      */
-    public static final Version DEFAULT_CRAM_VERSION = CRAM_v2_1;
+    public static final Version DEFAULT_CRAM_VERSION = CRAM_v3;
```

Comment:
Please add a test that verifies that the files we're creating now are actually v3 files, and that we can read them back in once we create one.